package.json

```diff
@@ -18,13 +18,23 @@
         "commands": [
             {
                 "command": "findComponentReferences",
-                "title": "Find Component References"
+                "title": "Find Component References",
+                "icon": "$(references)"
             },
             {
                 "command": "generateJsconfigAliases",
                 "title": "Generate Jsconfig Aliases"
             }
-        ]
+        ],
+        "menus": {
+            "editor/title": [
+                {
+                    "when": "editorLangId == 'vue' && veturExtended.findComponentReferences",
```

Comment:
See #6, when clause should be `enableFindReferencesButton` instead of `findComponentReferences`

Reply:
Thanks for correcting, messed up in namings a bit